etna/records/models.py

```diff
@@ -1,78 +1,316 @@
+        return candidate
+
+    def has_iaid(self) -> bool:
+        """
+        Returns `True` if a valid 'iaid' value can be extracted from the
+        raw data for this record. Otherwise `False`.
+        """
+        try:
+            self.iaid
+        except (ValueExtractionError, ValueError):
+            return False
+        else:
+            return True
+
+    @cached_property
+    def reference_number(self) -> str:
+        """
+        Return the "reference_number" value for this record (if one is available).
+
+        Raises `ValueExtractionError` when the raw data does not include
+        values in any of the expected positions.
+        """
+        try:
+            return self.template["referenceNumber"]
+        except KeyError:
+            pass
+        identifiers = self.get("identifier", ())
+        for item in identifiers:
+            try:
+                return item["reference_number"]
+            except KeyError:
+                pass
+        raise ValueExtractionError(
+            f"'reference_number' could not be extracted from source data: {self._raw}"
+        )
+
+    def has_reference_number(self) -> bool:
+        """
+        Returns `True` if a 'reference_number' value can be extracted from the
+        raw data for this record. Otherwise `False`.
+        """
+        try:
+            self.reference_number
+        except ValueExtractionError:
+            return False
+        else:
+            return True
+
+    @cached_property
+    def url(self):
+        """
+        Return the "url" value for this record. This value is typically
+        only present for 'interpretive' results from other websites.
+
+        Raises `ValueExtractionError` when the raw data does not include
+        values in any of the expected positions.
+        """
+        try:
+            return self.template["sourceUrl"]
+        except KeyError:
+            pass
+
+        locations = self.get("source.@location", ())
+        if len(locations) == 1:
+            return locations[0]["value"]
+        for item in locations:
+            if item.get("public", False):
+                return item["value"]
+        raise ValueExtractionError(
+            f"'url' could not be extracted from source data: {self._raw}"
+        )
+
+    def has_url(self) -> bool:
+        """
+        Returns `True` if a 'url' value can be extracted from the raw data
+        for this record. Otherwise `False`.
+        """
+        try:
+            self.url
+        except ValueExtractionError:
+            return False
+        else:
+            return True
+
+    @cached_property
+    def title(self) -> str:
+        try:
+            return self.template["summaryTitle"]
+        except KeyError:
+            return self.get("summary.title", default="")
+
+    @cached_property
+    def closure_status(self) -> str:
+        return self.get("access.conditions", default="")
```

Comment:
The path is `availability.access.condition.value` (existing code is incorrect) and template path is `@template.details.accessCondition` see `iaid=C7860693`
Perhaps using `return self.get("availability.access.condition.value", self.template.get("accessCondition", ""))`